Drive/rover_drive_system.hpp

```diff
@@ -244,27 +243,26 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
   enum class Modes : char
     {
       DriveMode= 'D',
-      SpinMode = 'S'
-      TranslateMode = 'T'
-      LeftWheelMode = 'L'
-      RightWheelMode = 'R'
+      SpinMode = 'S',
+      TranslateMode = 'T',
+      LeftWheelMode = 'L',
+      RightWheelMode = 'R',
       BackWheelMode = 'B'
-      
     };
 
   /// Stops the rover and sets a new mode.
   void SetMode()
   {
-    sjsu::LogWarning("Switching rover into %c mode...", mc_data_.drive_mode);
+    sjsu::LogWarning("Switching rover into %c mode...", current_drive_mode_);
     StopWheels();
-    switch (mc_data_.drive_mode)
+    switch (current_drive_mode_)
```

Comment:
This would mean I put the enum into the mission control struct in order to use the drive_mode variable

Reply:
Does it have to be? Can we not reuse the Modes type? I remember this being problematic in the past which is why we stuck with basic char type. Try declaring the Modes struct before the MissionControlData struct ?

Reply:
I just put the enum into the Mission Control data. It works fine but wasn't sure if that's what you wanted

Reply:
hmm normally i would say its fine but I think it might be best if we left it as public. Honestly I think it might be a good idea to move a lot of these structs to their own files. Cause now we have so many structs that we are having to recreate in different locations versus just importing the struct and just using it like a normal variable like char

Reply:
Yea that's probably a good idea. The issue is structs and chars cannot be compared or converted since they are not of the same type. Maybe I'll spend  more time on this so that it looks a lot cleaner and like you said use them in another file

Reply:
yeah this might be a bit tricky.